package.json

```diff
@@ -85,6 +85,7 @@
     "babel-eslint": "^8.0.0",
     "babel-plugin-external-helpers": "7.0.0-beta.2",
     "babel-plugin-transform-es2015-modules-strip": "^0.1.1",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
```

Comment:
Why not 7.0.0-beta.2 like the rest?

BTW I see now there's 7.0.0-beta.5 out, so I'll give it another go in another branch.

Reply:
I didn't choose the version number, I did : `npm i -D babel-plugin-transform-object-rest-spread` and that's all :smile:

Reply:
It seems it's the latest release : https://www.npmjs.com/package/babel-plugin-transform-object-rest-spread

Reply:
It's the latest stable. For every other plugin we are using the 7.x versions so we should do the same here.

Reply:
I'm having trouble updating to beta.5 anyway. I think we'll need to bug @hzoo so that we have all the plugins in sync...

Reply:
I found the 7.x release of this plugin :+1: